### PR TITLE
Spring cleaning

### PR DESCRIPTION
**Pull request by Rulasmur**
# Description
> Went through and cleaned up some stuff as per checkstyle and SonarLint

# Testing
> Nothing to test

### Pre Pull Request Checklist:
- [x] Code has been scanned with [SonarLint](http://www.sonarlint.org/intellij/)
- [x] Methods have been annotated with @ Nullable and @ Nonnull ([Read more](https://github.com/MovingBlocks/DestinationSol/wiki/@Nullable-&-@Nonnull-Quickstart))
- [x] There are no errors present in the project
- [x] Code has been formatted and indented
- [x] Methods have appropriate Javadoc ([How to write Javadoc](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html))

### CODE REVIEW

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

File: launcher/lwjre/lib/calendars.properties [hunk 1]
```diff
@@ -21,40 +21,36 @@
 #
```
Comment (rzats) [inline launcher/lwjre/lib/calendars.properties]: Are you *sure* you want to touch this (or anything else in the lwjre directory)? This is a) not created by us and b) some sort of weird config format that might be broken by wrong spacing

File: main/imgSrcs/asteroids/pack.json [hunk 2]
```diff
@@ -2,5 +2,5 @@
   minWidth: 512,
```
Comment (rzats) [inline main/imgSrcs/asteroids/pack.json]: I would leave the JSON configs alone, too. They aren't _actual_ JSON anyways and use a custom parser that might be broken by the lack of trailing commas.

File: main/src/org/destinationsol/common/SolMath.java [hunk 3]
```diff
@@ -44,6 +43,7 @@ protected Vector2 newObject() {
     };
     public static int VECTORS_TAKEN;
     public static Vector2 tmp = new Vector2();
+    private static Logger logger = LoggerFactory.getLogger(SolMath.class);
```
Comment (rzats) [inline main/src/org/destinationsol/common/SolMath.java]: Shouldn't this always be at the top?

File: launcher/lwjre/lib/calendars.properties (hunk 1 above)
Comment (Cervator) [inline launcher/lwjre/lib/calendars.properties]: I suspect maybe a search term for files of a particular type dug up stuff under launcher - yeah IMHO everything in there should be ignored (and later on extracted)

File: main/imgSrcs/asteroids/pack.json (hunk 2 above)
Comment (Cervator) [inline main/imgSrcs/asteroids/pack.json]: Besides now we get to translate it all into proper assets! :D

File: launcher/lwjre/lib/calendars.properties [hunk 4]
```diff
@@ -21,40 +21,36 @@
...
 # Japanese imperial calendar
 #
 #   Meiji  since 1868-01-01 00:00:00 local time (Gregorian)
 #   Taisho since 1912-07-30 00:00:00 local time (Gregorian)
 #   Showa  since 1926-12-25 00:00:00 local time (Gregorian)
 #   Heisei since 1989-01-08 00:00:00 local time (Gregorian)
-calendar.japanese.type: LocalGregorianCalendar
-calendar.japanese.eras: \
+calendar.japanese.type:LocalGregorianCalendar
+calendar.japanese.eras:\
```
Comment (Rulasmur) [inline launcher/lwjre/lib/calendars.properties]: Auto format seems to like it like that

File: launcher/lwjre/lib/hijrah-config-umalqura.properties [hunk 5]
```diff
@@ -52,19 +52,14 @@
 #    * Each month should be 29 or 30 days long.
 #    * There must be one or more space characters between the months.
 #
-
 # Version of this definition
 version=1.8.0_1
-
 # Java chronology ID
```
Comment (Rulasmur) [inline launcher/lwjre/lib/hijrah-config-umalqura.properties]: Its just auto format doing its thing

**Pull request closed without merging**